java/runtime/src/main/java/io/ray/runtime/util/IdUtil.java

```diff
@@ -75,4 +78,18 @@ private static long murmurHash64A(byte[] data, int length, int seed) {
 
     return h;
   }
+
+  /**
+   * Compute the actor ID of the task which created this object.
+   * @return The actor ID of the task which created this object.
+   */
+  public static ActorId getActorIdFromObjectId(ObjectId objectId) {
+    byte[] taskIdBytes = new byte[TaskId.LENGTH];
+    System.arraycopy(objectId.getBytes(), 0, taskIdBytes, 0, TaskId.LENGTH);
+    TaskId taskId = TaskId.fromBytes(taskIdBytes);
+    byte[] actorIdBytes = new byte[ActorId.LENGTH];
+    System.arraycopy(taskId.getBytes(), TaskId.UNIQUE_BYTES_LENGTH,
```

Comment:
You can get actor ID from object ID directly. No need to convert it to task ID first.

Reply:
Oh, I see. the actor ID part is not at the end of object ID. So at least you can copy only once.

Reply:
I still prefer the current modification method, which makes the code more readable. There is a clear process of Object ID -> Task ID > Actor ID.